homeassistant/components/blink/__init__.py

```diff
@@ -84,28 +85,29 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     auth_data = deepcopy(dict(entry.data))
     blink.auth = Auth(auth_data, no_prompt=True, session=session)
     blink.refresh_rate = entry.options.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
+    coordinator = BlinkUpdateCoordinator(hass, blink)
 
     try:
-        await blink.start()
+        await coordinator.api.start()
```

Comment:
I think you can keep this at `blink` btw

Reply:
I think it's fine as blink too, but we moved everything else to `coordinator.api` - I don't think it costs any more or less from execution time... Should I change it back?  Should I change all other refs in the routine to `blink.xxx` rather than `coordinator.api.xxx` - To me it doesn't make sense to be half one way and half the other....

Reply:
I think it does cost a bit more since you're accessing another variable. but this isn't code we run a lot so I am not sure what is best here